guides/development-workflow.md

```diff
@@ -190,6 +190,7 @@ To deploy:
   ```
   git fetch
   git checkout master
+  git pull
```

Comment:
@dgmstuart My Tower merges are configured to use `--no-ff` and my CLI is default.

I believe we do only want to merge into the latest version of master.

Normally to get others work I fetch then checkout the remote. When somebody pushes more to that branch I will fetch, and pull.

I agree, we shouldn't have to rely on on personal configurations being made.